ister_gui.py

```diff
@@ -1423,6 +1423,7 @@ def _static_configuration(self, _):
         try:
             subprocess.call(['/usr/bin/systemctl', 'restart',
                              'systemd-networkd', 'systemd-resolved'])
+            time.sleep(.1)
```

Comment:
What about just a `systemctl status systemd-networkd systemd-resolved` to make sure both are up? That seems easier than parsing ip a output to me.